src/pages/groupComparison/GroupComparisonMutationsTabPlot.tsx

```diff
@@ -124,7 +124,10 @@ export default class GroupComparisonMutationsTabPlot extends React.Component<
                             plotLollipopTooltipCountInfo={
                                 plotLollipopTooltipCountInfo
                             }
-                            axisMode={this.props.store.userSelectedAxisMode}
+                            axisMode={
+                                this.props.store.userSelectedAxisMode ||
```

Comment:
sorry to be annoying but i think we should probably put this default inside of the the userSelectedAxisMode.

And also, I think we can just call it ```axisMode``` since I don't think we strongly need to distinguish it from a default or something else.